bin2c/bin2c.c

```diff
@@ -96,33 +114,16 @@ int main(int argc, char *argv[])
     minor = fileLen % 16;
 
     bufptr = buffer;
-
+	
     for (i = 0; i < major; i++)
     {
-        fprintf(outputFile, "%u,%u,%u,%u,%u,%u,%u,%u,%u,%u,%u,%u,%u,%u,%u,%u",
-            bufptr[0], bufptr[1], bufptr[2], bufptr[3], bufptr[4], bufptr[5], bufptr[6], bufptr[7],
-            bufptr[8], bufptr[9], bufptr[10], bufptr[11], bufptr[12], bufptr[13], bufptr[14], bufptr[15]);
-
-        if ((i == major - 1) && !minor)
-        {
-            fprintf(outputFile, "\n");
-        }
-        else {
-            fprintf(outputFile, ",\n");
-        }
-
-        bufptr += 16;
+		bufptr = fprintList(outputFile, bufptr, 16, useHex? H_OUTPUT_FORMAT_STR: D_OUTPUT_FORMAT_STR);
```

Comment:
Typically not a fan of ternaries inline with function parameters - as this is duplicated below, consider assigning to a var first.